xwiki-platform-core/xwiki-platform-menu/xwiki-platform-menu-ui/src/main/resources/Menu/UIExtensionSheet.xml

```diff
@@ -56,12 +56,17 @@
   ## Prepare the extension point map.
   #set ($menuPanel = "{{velocity}}
     ${escapetool.h}panelheader(""$doc.plainTitle"")
-    {{menu type=""vertical collapsible open""}}{{include reference=""${escapetool.d}uix.doc.documentReference"" /}}{{/menu}}
+    {{menu type=""vertical collapsible open""
+      label="$escapetool.velocity($services.rendering.escape($doc.plainTitle, 'xwiki/2.1'))"}}{{include
```

Comment:
Shouldn't this be double quotes (I mean `""`) as otherwise this would end the string that starts at `{{velocity}}`?